### PR TITLE
Display changes

### LINKED ISSUE #37: Display messages
Right now there is nothing implemented to display messages. Plan is to define them as toasts in fututre

### PR DESCRIPTION
**Pull request by szewczykmira**
Changes of landing page and rest of website.
Closes #37

### CODE REVIEW

File: conferences/templates/conferences/about.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+{% load static %}
+
+<div class="grey-text text-darken-1">
+    <div class="container">
+        <div class="row flexbox center-items-horizontal">
+            <div class="col l7 m5 s12">
+                <div class="text-block text-large">
+                    <b>ZOSIA</b>, czyli <b>Zimowy Obóz Studentów Informatyki A</b>, to impreza organizowana przez Koło Studentów Informatyki od ponad 20 lat.
```
Comment (scoiatael) [inline conferences/templates/conferences/about.html]: English?

File: conferences/templates/conferences/index.html [hunk 2]
```diff
@@ -6,80 +6,26 @@
...
-          </div>
-        </div>
-    </div>
-</div>
+{% include "conferences/about.html" %}
+<!-- MAP AND ADDRESS -->
+{% include "conferences/zosia.html" %}
 <!-- BLOG? -->
+<!-- ORGANIZATORS -->
+{% include "conferences/organizators.html" %}
```
Comment (scoiatael) [inline conferences/templates/conferences/index.html]: organizers?

File: zosia16/settings/common.py [hunk 3]
```diff
@@ -147,10 +147,12 @@ def random_string(length=10):
 
 USE_I18N = True
 
-USE_L10N = True
+USE_L10N = False
```
Comment (scoiatael) [inline zosia16/settings/common.py:150]: why?

File: zosia16/urls.py [hunk 4]
```diff
@@ -24,4 +26,4 @@
     url(r'^blog/', include('blog.urls')),
     url(r'^sponsors/', include('sponsors.urls')),
     url(r'^lectures/', include('lectures.urls')),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (scoiatael) [inline zosia16/urls.py:30]: Can you double check it doesn't serve static when DEBUG=False? And add a note if it really does not.

File: zosia16/settings/common.py (hunk 3 above)
Comment (scoiatael) [reply zosia16/settings/common.py:150]: Hm, standard date display. Not sure if world is ready for that ;)

Comment (szewczykmira) [reply zosia16/settings/common.py:150]: As I show you - date i10n could look really awful :)

**Comment by szewczykmira** [comment]
@scoiatael - should I change file name from `organizators.html` to `organizers.html`?

**Comment by scoiatael** [comment]
@szewczykmira yep, would be awesome if you had time to :)

**Review by scoiatael — APPROVED** [review]

**Pull request merged**